.github/workflows/tests.yml

```
@@ -205,6 +205,10 @@ jobs:
       LXD_CEPH_CLUSTER: "ceph"
       LXD_CEPH_CEPHFS: "cephfs"
       LXD_CEPH_CEPHOBJECT_RADOSGW: "http://127.0.0.1"
+      LXD_OVN_NB_CONNECTION: "ssl://127.0.0.1:6641"
```

```suggestion
      LXD_OVN_NB_CONNECTION: "ssl:127.0.0.1:6641"
```
Sorry for that one, I wrongly added `//` :/